zkstack_cli/crates/zkstack/build.rs

```diff
@@ -19,3 +29,50 @@ fn main() -> eyre::Result<()> {
     .unwrap();
     Ok(())
 }
+
+fn copy_completion_scripts() -> anyhow::Result<()> {
+    let crate_name = env!("CARGO_PKG_NAME");
+
+    // Create local config directory
+    let local_config_dir = config_local_dir().unwrap().join(crate_name);
+    std::fs::create_dir_all(&local_config_dir)?;
+
+    // Array of supported shells
+    let shells = ["bash", "zsh"];
```

Comment:
do we have a special case for fish? (usually it's the most problematic one)

Reply:
Added support for fish in https://github.com/matter-labs/zksync-era/pull/3097/commits/61ac014f7747429edefc275108337b0535fd1df5